src/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+init();
+
+function init() {
+  const clock = document.querySelector('.clock');
+  const widthClock = window.getComputedStyle(clock, null).width;
+
+  for (let i = 0; i < 11; i++) {
+    const div = document.createElement('div');
+    div.classList.add(`scale`);
+    clock.prepend(div);
+  }
+
+  const scale = clock.querySelectorAll('.scale');
+  drawScaleClock(scale, widthClock);
+  drawScaleNumClock(scale);
+}
+
+function drawScaleClock(scale, widthClock) {
+  const width = parseInt(widthClock, 10);
+  const delta = Math.PI * 2 / scale.length;
+  let angle = 0;
+  let angleScale = 60;
+
+  for (let i = 0; i < scale.length; i++) {
+    scale[i].style.top
+      = 100 * Math.sin(angle) * (width / 235) + (width / 2.22) + 'px';
+    scale[i].style.left
+      = 100 * Math.cos(angle) * (width / 235) + (width / 2) + 'px';
+    scale[i].style.transform
+      = `translateX(-50%) rotate(${angleScale += 30}deg)`;
+
+    angle += delta;
+  }
+}
+
+function drawScaleNumClock(scale) {
+  for (let i = 0; i <= scale.length; i++) {
```

Comment:
```suggestion
  for (let i = 0; i <= scale.length; i += 3) {
```
тоді switch case немає сенсу всередині